lms/djangoapps/course_api/serializers.py

```diff
@@ -14,9 +14,9 @@ class _MediaSerializer(serializers.Serializer):  # pylint: disable=abstract-meth
     """
     Nested serializer to represent a media object.
     """
-    def __init__(self, uri_attribute, *args, **kwargs):
+    def __init__(self, *args, **kwargs):
+        self.uri_attribute = kwargs.pop("uri_attribute", None)
```

Comment:
Is this part of the Swagger configuration? I'm skipping it in my initial attempt to reproduce these changes for Swagger.